pkg/controllers/provisioning/suite_test.go

```diff
@@ -71,7 +69,7 @@ var _ = BeforeSuite(func() {
 	fakeClock = clock.NewFakeClock(time.Now())
 	cluster = state.NewCluster(fakeClock, env.Client, cloudProvider)
 	nodeController = informer.NewNodeController(env.Client, cluster)
-	prov = provisioning.NewProvisioner(env.Client, corev1.NewForConfigOrDie(env.Config), events.NewRecorder(&record.FakeRecorder{}), cloudProvider, cluster)
+	prov = provisioning.NewProvisioner(env.Client, corev1.NewForConfigOrDie(env.Config), cloudProvider, cluster)
```

Comment:
I think we may need to inject a recorder into the context generally for all of our test environments. If we don't pass it through, I think these tests will panic when we try to fire errors

Reply:
ya thats what thinkered me